docs/docs/20-usage/50-environment.md

```diff
@@ -57,7 +57,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_REPO_SCM`                    | repository SCM (git)                                                                         |
 | `CI_REPO_URL`                    | repository web URL                                                                           |
 | `CI_REPO_CLONE_URL`              | repository clone URL                                                                         |
-| `CI_REPO_DEFAULT_BRANCH`         | repository default branch (master)                                                           |
+| `CI_REPO_DEFAULT_BRANCH`         | repository default branch (main)                                                           |
```

Comment:
This needs to ve changed in code too

Reply:
Isn't it changed in this PR already 🤔

Reply:
https://github.com/woodpecker-ci/woodpecker/blob/524611cf00ea9cc86e2e548244937ec9b7141b24/server/forge/gitlab/convert.go#L107
https://github.com/woodpecker-ci/woodpecker/blob/524611cf00ea9cc86e2e548244937ec9b7141b24/server/forge/gitlab/convert.go#L56-L58
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L100-L102
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L159-L161
https://github.com/woodpecker-ci/woodpecker/blob/1b43b0bf20c03e63fb270453d2ffa5afed456fb2/server/forge/github/convert.go#L26
https://github.com/woodpecker-ci/woodpecker/blob/6d2240b2e6f844f267866a26d45b1faac462788c/server/forge/bitbucket/convert.go#L62

Reply:
-> #2057